docs-source/content/faq/oci-lb.md

```diff
@@ -9,19 +9,19 @@ If you are running your Kubernetes cluster on Oracle Container Engine
 for Kubernetes (commonly known as OKE), you can have OCI automatically
 provision load balancers for you by creating a `Service` of type
 `LoadBalancer` instead of (or in addition to) installing an
-ingress controller like Traefik or Voyager.
+Ingress controller like Traefik or Voyager.
 
 OKE Kubernetes worker nodes normally do not have public IP addresses.
 This means that the `NodePort` services created by the operator are
 not usable, because they would expose ports on the worker node's private
 IP addresses only, which are not reachable from outside the cluster.  
-Instead, you can use an OCI Load Balancer to provide access
+Instead, you can use an OCI load balancer to provide access
 to services running in OKE.
 
 {{% notice note %}}
-It is also possible, if desirable, to have an OCI Load Balancer route
-traffic to an ingress controller running inside the Kubernetes cluster
-and have that ingress controller in turn route traffic to services in the
+It is also possible, if desirable, to have an OCI load balancer route
+traffic to an Ingress controller running inside the Kubernetes cluster
```

Comment:
I think "Ingress Controller" might be more correct for this

Reply:
Hmmm. We don't capitalize controller anywhere else in the docs, except where it is part of a product name, such as Voyager Ingress Controller. In the Kubernetes docs, it's not capitalized (and Ingress is not consistently capitalized! Sheesh). Still want me to capitalize these two instances of controller or all of them?

Reply:
ok that's fine, leave it as is